_data/roadmap.yml

```diff
@@ -63,17 +63,29 @@
     - name: GUI out of beta
       date:
       status: ongoing
-    - name: Kovri Alpha release
+    - GUI port to android
       date:
       status: ongoing
-    - name: Website Redesign
+    - name: Kovri alpha release
+      date:
+      status: ongoing
```

Comment:
@anonimal can I change the value to `s/ongoing/upcoming/` and it will look ok on the page?